src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -318,7 +318,7 @@ function IOURequestStepScan({
             // If the user started this flow using the Create expense option (combined submit/track flow), they should be redirected to the participants page.
             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
             if ((transaction?.isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK && !report?.reportID) || iouType === CONST.IOU.TYPE.CREATE) {
-                if (activePolicy && isPaidGroupPolicy(activePolicy) && !shouldRestrictUserBillableActions(activePolicy.id)) {
+                if (activePolicy && isPaidGroupPolicy(activePolicy) && activePolicy?.isPolicyExpenseChatEnabled && !shouldRestrictUserBillableActions(activePolicy.id)) {
```

Comment:
hmmmm isn't this code wrong (even before your changes)? Why is it using activePolicy instead of the policy of the chat you are requesting money from?
Same in the other file really.

Reply:
What do you mean? These components are not used when you click on the + button in the workspace chat?

Reply:
That's a whole PR where exactly is the explanation?
Also is your answer to my previous question `no`?

Reply:
Oh, but this only runs if `isFromGlobalCreate` OR when `iouType === CONST.IOU.TYPE.CREATE`, I am not sure what that last condition means, do you?